emcc.py

```diff
@@ -1313,14 +1316,10 @@ def check(input_file):
       if final_suffix in EXECUTABLE_ENDINGS:
         diagnostics.warning('emcc', '-shared/-r used with executable output suffix. This behaviour is deprecated.  Please remove -shared/-r to build an executable or avoid the executable suffix (%s) when building object files.' % final_suffix)
       else:
+        if options.shared:
+          diagnostics.warning('emcc', 'linking a library with `-shared` will emit a static object file.  This is form of emulation to support existing build system.  If you want to build a runtime shared library use the SIDE_MODULE setting.')
```

Comment:
```suggestion
          diagnostics.warning('emcc', 'linking a library with `-shared` will emit a static object file.  This is a form of emulation to support existing build systems.  If you want to build a runtime shared library use the SIDE_MODULE setting.')
```
I thought I already commented about this... was it irrelevant and resolved for some reason?